<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingBufferViewer</class>
 <widget class="QDialog" name="RingBufferViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ring buffer viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ImageWidget" name="imageWidget">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="indexSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="indexBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QSplitter</extends>
   <header>guiqwt.plot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>indexSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
